docs/book/v4/migration/v3-to-v4.md

```diff
@@ -1,5 +1,3 @@
 # Migration from Version 3 to 4
 
-INFO: **Work in progress**
-We are currently working on the migration guide.
-We will update this page as soon as possible!
+The ServiceManager 4 introduce new interfaces and updated the factory class structure. We have [Rector Rule](https://github.com/laminas/laminas-servicemanager-migration) migrations that cover that.
```

Comment:
It does not look as if all the changes have been covered: https://github.com/laminas/laminas-servicemanager/releases/tag/4.0.0